.github/workflows/e2e-progress-tracker.yml

```
@@ -0,0 +1,216 @@
+          node-version: '18'
+          cache: 'npm'
+          cache-dependency-path: 'web/package-lock.json'
+
+      - name: Install dependencies
+        working-directory: web
+        run: npm ci
+
+      - name: Run E2E tests and capture results
+        working-directory: web
+        continue-on-error: true
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:8:42: Double quote to prevent globbing and word splitting

(shellcheck)